tests/run-tests

```diff
@@ -49,15 +49,15 @@ def convert_regex_escapes(line):
     return bytes(''.join(cs), 'utf8')
 
 
-def run_micropython(pyb, args, test_file):
+def run_micropython(pyb, args, test_file, is_special=False):
     special_tests = ('micropython/meminfo.py', 'basics/bytes_compare3.py', 'basics/builtin_help.py', 'thread/thread_exc2.py')
-    is_special = False
     if pyb is None:
         # run on PC
         if test_file.startswith(('cmdline/', 'feature_check/')) or test_file in special_tests:
```

Comment:
This check for "feature_check" becomes redundant with the "is_special" arg passed.